src/community/design-system-working-group/index.md.njk

```diff
@@ -14,6 +14,15 @@ The working group:
 - makes recommendations to help contributors improve their work
 - advocates for the needs of colleagues across government
 
+The working group reviews:
+
+- new patterns and components
+- changes to components that will easily be recognised
```

Comment:
I know we mean it's the changes that will be recognised, but someone might think it's the components we're referring to. Would "component changes"work?

Can we say who will recognise the changes and make this active? (So, "component changes that users will easily recognise," or similar.)